_updates/2020-07-01-july.md

```diff
@@ -0,0 +1,68 @@
+---
+title: July 2020
+---
+
+## Brick by brick 🧱
+
+As we get closer and closer to staring [SNaaS](https://docs.sensenet.com/concepts/introduction/04-what-is-snaas), instead of hot new features stabilization and simplification came to the fore. We are working hard on making the core of the admin surface super fast, super effective and easy-to-use. Followings are the updates we've made in the last 4 weeks:
```

Comment:
```suggestion
As we get closer and closer to starting [SNaaS](https://docs.sensenet.com/concepts/introduction/04-what-is-snaas), instead of hot new features stabilization and simplification came to the fore. We are working hard on making the core of the admin surface super fast, super effective and easy-to-use. Followings are the updates we've made in the last 4 weeks:
```